Editor/Waiter.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Threading.Tasks;
+using AmazonGameLiftPlugin.Core.DeploymentManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared;
+
+namespace AmazonGameLift.Editor
+{
+    internal class Waiter
+    {
+        public event Action InfoUpdated;
+
+        private readonly CoreApi _gameLiftCoreApi;
+        private bool _isWaiting;
+
+        public Waiter(CoreApi coreApi)
+        {
+            _gameLiftCoreApi = coreApi;
```

Comment:
Rename, `GameLiftCoreApi` and `CoreApi` are separate things so this is confusing

Reply:
renamed